### ISSUE TITLE
push users towards tags

### ISSUE THREAD

**Issue by coke** [tadzik/rakudobrew#80]
Post Christmas, we want users to probably grab the latest tag, not nom, especially if nom isn't where we cut from for each release.  (e.g. right now 2016.01.1 is the latest, but the default is to get nom, which reports as 2015.12 still)

**Comment by stmuk** [comment]
see  https://github.com/tadzik/rakudobrew/pull/75

Maybe needs tweaking to remove hyphen from "build-release" and cleanly apply again.

**Comment by stmuk** [comment]
actually nom:VERSION doesn't work anymore and I can't think of a simple way of scripting "most recent release"

**Comment by stmuk** [comment]
apparently git show nom:VERSION will show the last release once the release is merged back into nom so maybe this mechanism will work

**Comment by coke** [comment]
We shouldn't rely on nom going forward. We can sort the tags to show most
recent...

On Tuesday, February 2, 2016, Steve Mynott <notifications@github.com> wrote:

> apparently git show nom:VERSION will show the last release once the
> release is merged back into nom so maybe this mechanism will work
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tadzik/rakudobrew/issues/80#issuecomment-178809173>.
>


-- 
Will "Coke" Coleda

**Comment by stmuk** [comment]
If you mean the script is supposed to retrieve all the tags and sort them
then that won't scale over time since the number of tags will increase and
the process slow down.

There has to be some sort of simple pointer to flag the last stable release
so that tools can pull it automatically.

On 3 February 2016 at 16:01, Will Coleda <notifications@github.com> wrote:

> We shouldn't rely on nom going forward. We can sort the tags to show most
> recent...
>
> On Tuesday, February 2, 2016, Steve Mynott <notifications@github.com>
> wrote:
>
> > apparently git show nom:VERSION will show the last release once the
> > release is merged back into nom so maybe this mechanism will work
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/tadzik/rakudobrew/issues/80#issuecomment-178809173>.
> >
>
>
> --
> Will "Coke" Coleda
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tadzik/rakudobrew/issues/80#issuecomment-179311719>.
>



-- 
4096R/EA75174B Steve Mynott <steve.mynott@gmail.com>

**Comment by coke** [comment]
A simple text file on rakudo.org, perhaps?

On Wednesday, February 3, 2016, Steve Mynott <notifications@github.com>
wrote:

> If you mean the script is supposed to retrieve all the tags and sort them
> then that won't scale over time since the number of tags will increase and
> the process slow down.
>
> There has to be some sort of simple pointer to flag the last stable release
> so that tools can pull it automatically.
>
> On 3 February 2016 at 16:01, Will Coleda <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
> > We shouldn't rely on nom going forward. We can sort the tags to show most
> > recent...
> >
> > On Tuesday, February 2, 2016, Steve Mynott <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> > wrote:
> >
> > > apparently git show nom:VERSION will show the last release once the
> > > release is merged back into nom so maybe this mechanism will work
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/tadzik/rakudobrew/issues/80#issuecomment-178809173
> >.
> > >
> >
> >
> > --
> > Will "Coke" Coleda
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/tadzik/rakudobrew/issues/80#issuecomment-179311719>.
> >
>
>
>
> --
> 4096R/EA75174B Steve Mynott <steve.mynott@gmail.com
> <javascript:_e(%7B%7D,'cvml','steve.mynott@gmail.com');>>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tadzik/rakudobrew/issues/80#issuecomment-179337925>.
>


-- 
Will "Coke" Coleda

**Comment by garu** [comment]
So umm... I'm probably missing something from this discussion and about to make a fool out of myself, and this is an old thread so maybe it's already resolved, but I'm going to say it anyway: you do know git allows for multiple tags on the same commit, right? Why not simply add a 'latest', 'stable', or even 'latest-stable' tag to the target commit?

**Comment by tadzik** [comment]
@garu I like this idea, I'll take it to the rakudo team to talk about it :)

**Comment by coke** [comment]
I'm pretty sure we don't have to have tags in the rakudo repository that move.

**Comment by patzim** [comment]
We display versions sorted reversed so latest are lowest. That's working well enough I'd say. Please reopen if that's not good enough for someone.
